packages/trader/src/App/Components/Elements/ContractAudit/contract-details.jsx

```diff
@@ -108,15 +108,13 @@ const ContractDetails = ({ contract_end_time, contract_info, duration, duration_
                     label={localize('Start time')}
                     value={toGMTFormat(epochToMoment(purchase_time)) || ' - '}
                 />
-                {!isDigitType(contract_type) && (
-                    <ContractAuditItem
-                        id='dt_entry_spot_label'
-                        icon={<Icon icon='IcContractEntrySpot' size={24} />}
-                        label={localize('Entry spot')}
-                        value={addCommaToNumber(entry_spot_display_value) || ' - '}
-                        value2={toGMTFormat(epochToMoment(entry_tick_time)) || ' - '}
-                    />
-                )}
+                <ContractAuditItem
```

Comment:
In this case, we need to display entry spot_markers on the chart as well.
<img width="176" alt="Screenshot 2022-05-30 at 12 00 22 PM" src="https://user-images.githubusercontent.com/70938039/170914873-280d6043-2856-4332-aac1-84ca6015bd03.png">